Atos20/indecision-maker-server
# Board game setup bd

### PR Template:
#### What's this PR do?  

- Adds board_game schema, seed, and datafile
- Seeds boardgame data through the boardgameatlas API instead of having to hardcode
- Easily dynamic to adjust for more/less or different boardgames( check out the games array, its as simple as just adding names to that)

#### Where should the reviewer start?  

- Start by checking out the logic on the data population to see if it can be cleaned up at all

#### How should this be manually tested?  

- Pull down the local changes
- Run `knex migrate:latest` and `knex seed:run` to populate the databse
- Check out the data that is entered now under the board_games table through Postico or other means

#### Any background context you want to provide?  
- The API doesn't have a crazy amount of information, but if we are hindering on someone already owning a boardgame do we really need to provide much other than the basics?
- Some broken data for a couple of the games, may need to filter those out in favor of ones who have all the data
- Feel free to adjust any of the boardgame names, pretty much just tried to find the most popular ones I could think of
 
#### What are the relevant tickets?  
 closes #10 

#### Screenshots (if appropriate) 

![Screen Shot 2020-10-28 at 11 47 33 PM](https://user-images.githubusercontent.com/65369275/97523183-f5c40900-1977-11eb-8b1e-647d872bfdf1.png)
